rmw_connext_cpp/src/functions.cpp

```diff
@@ -934,6 +934,47 @@ rmw_create_client(
     goto fail;
   }
 
+  // For now, use the default QoS profile for services.
```

Comment:
I'd move this code to `create_requester`, pass a `rmw_qos_profile_t` to `rmw_create_client` and default it to `rmw_qos_profile_services_default`.